internal/config/loader/loaderutils/files.go

```diff
@@ -0,0 +1,33 @@
+package loaderutils
```

Comment:
Create `loaderutils` so that server packages wouldn't leak into client packages, like they were when client was importing `loader` directly.